cmd/spire-agent/cli/api/fetch_cli.go

```diff
@@ -122,6 +123,13 @@ func (f FetchCLI) writeResponse(resp *workload.X509SVIDResponse) error {
 		if err != nil {
 			return err
 		}
+
+		for _, trustDomain := range svid.FederatesWith {
+			err = f.writeCerts(federatedBundleName, resp.FederatedBundles[trustDomain])
```

Comment:
won't this overwrite the file one each iteration if there is more than one trust domain being federated with?

Reply:
Good point!